src/avt/Database/Ghost/avtUnstructuredDomainBoundaries.C

```diff
@@ -794,8 +819,9 @@ avtUnstructuredDomainBoundaries::ExchangeMaterial(vector<int>    domainNum,
 {
     bool haveMixedMaterials = false;
     for (size_t i = 0; i < domainNum.size(); i++)
-        if (mats[i]->GetMixlen() != 0)
-            haveMixedMaterials = true;
+        if (mats[i] == NULL)
```

Comment:
Looks like this check should be if mats[i] != NULL?
And you can combine the ifs into one statement with &&

Reply:
Good catch! Fixed.